include/date/date.h

```diff
@@ -91,6 +91,11 @@ namespace date
 // Configuration |
 //---------------+
 
+#if defined(RTC) //RTC Configuration
+#    define INSTALL ""//This is used to ensure some codeblocks in tz.h and tz.cpp activate.
+#    define USE_OS_TZDB 1 //This is used to ensure that we use the compiled TZDB format, not the more complex uncompiled version.
+#endif
```

Comment:
Since warnings as errors isn't an issue for 3rdparty, warnings are not a big deal. You can also remove this warning with `#    undef USE_OS_TZDB` right before it.